### ISSUE TITLE
Recommendation doesn't work on pages

### ISSUE THREAD

**Issue by jr-dig-orgel** [EEXCESS/wordpress-plugin#9]
In our installation get see the EEXCESS Button when we do a new blog post and also get recommendations.
When we edit a page in wordpress, we also see the EEXCESS button, but we don't get recommendations. 
Maybe the reason is that on the edit page the eexcess_container is there on the edit page site. The screenshot shows the missing container.
![wordpress-eexcess](https://cloud.githubusercontent.com/assets/4506957/11266206/24ac9576-8ea3-11e5-8cdb-62bf7eb99bfa.PNG)

**Comment by n-witt** [comment]
Which version did you install? It's recommended to use this:
https://wordpress.org/plugins/eexcess/

**Comment by jr-dig-orgel** [comment]
Yes, we have installed the version from the wordpress.org plugins directory.
